policy/resolver.go

```diff
@@ -710,6 +719,42 @@ func (s *LocalServices) refreshChecksums(executionJob *ExecutionJob, collectorJo
 	}
 }
 
+func (s *LocalServices) collectRisks(ctx context.Context, cache *resolverCache, policyMrn string) error {
+	policyObj, ok := cache.bundleMap.Policies[policyMrn]
+	if !ok || policyObj == nil {
+		return errors.New("cannot find policy '" + policyMrn + "' while resolving")
+	}
+
+	for _, g := range policyObj.Groups {
+		for _, p := range g.Policies {
+			if err := s.collectRisks(ctx, cache, p.Mrn); err != nil {
+				return err
+			}
+		}
+	}
+
+	for _, rf := range policyObj.RiskFactors {
+		s.mergeRisk(cache, rf)
+	}
+
+	return nil
+}
+
+func (s *LocalServices) mergeRisk(cache *resolverCache, riskFactor *RiskFactor) {
+	if existing, ok := cache.riskFactors[riskFactor.Mrn]; ok {
+		if riskFactor.Magnitude != nil {
+			existing.Magnitude = riskFactor.Magnitude
+		}
+
+		if riskFactor.Action != explorer.Action_UNSPECIFIED {
+			existing.Action = riskFactor.Action
```

Comment:
Perhaps an edge case:
Could the same risk factor be defined in multiple policies?
This line would then mean that the last non-unspecified action wins. Is this as it should be?

Reply:
ya, i think that problem already exists. But if its defined in the space policy it will win